flytekit/core/array_node.py

```diff
@@ -79,16 +79,23 @@ def __init__(
 
         # TODO - bound inputs are not supported at the moment
         self._bound_inputs: Set[str] = set()
+        self._excluded_inputs: Set[str] = set()
+        if isinstance(target, (LaunchPlan, FlyteLaunchPlan)):
+            # rely on user defined workflow interface since we don't fetch reference launch plans from admin
+            if not isinstance(target, ReferenceLaunchPlan):
```

Comment:
What is expected to happen when the reference launch plan references a launch plan with fixed_inputs?